CriticalMass/AppController.swift

```diff
@@ -26,11 +26,15 @@ class AppController {
     private var dataStore = MemoryDataStore()
 
     private lazy var requestManager: RequestManager = {
-        RequestManager(dataStore: dataStore, locationProvider: LocationManager(), networkLayer: networkOperator, idProvider: idProvider, url: Constants.apiEndpoint)
+        RequestManager(dataStore: dataStore, locationProvider: LocationManager(), networkLayer: networkOperator, idProvider: idProvider)
     }()
 
     private let networkOperator: NetworkOperator = {
-        NetworkOperator(networkIndicatorHelper: NetworkActivityIndicatorHelper())
+        let configuration = URLSessionConfiguration.default
+        configuration.requestCachePolicy = .reloadIgnoringLocalCacheData
```

Comment:
While adding an error view I thought we might need to set a timeout timeinterval as well. But since it is not related to this PR not requested here. Just thought we could discuss here

Reply:
Doesn’t the default configuration come with a timeout?

Reply:
No. E.g the Twitter Feed was loading all the time when the BE was not responding